pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -379,9 +387,12 @@ def spawn_ddp_children(self, model):
         os.environ['NODE_RANK'] = node_rank
         os.environ['LOCAL_RANK'] = '0'
 
+        # when user is using hydra find the absolute path
+        path_lib = abspath if not HYDRA_AVAILABLE else to_absolute_path
```

Comment:
This will require Hydra 1.0.0rc1 (pre-release) or newer to work correctly.
In previous version to_absolute_path will throw an exception if Hydra is not initialized.
To be defensive, you should fallback to abspath if to_absolute_path throws.

Reply:
any script for lightning using argparse can take in `--gpus x`.
but now that hydra uses a different convention it makes it hard for us to support those scripts since we can't pass in `--gpus x`.

The flag is used in `Trainer(gpus=x)`
and users define it it their argparse normally or use `Trainer.from_argparse_args` which does it for them...